### ISSUE TITLE
Zoom cookies expire after a day

### ISSUE THREAD

**Issue by 1337w0rm** [1337w0rm/YeetMeet#9]
Zoom cookies stored in pkl files get expired after 24 hours and the user have to relogin again to use it.

If anybody could figure out a workaround for this, such that the  cookies gets refreshed automatically and the user don't have to relogin again after 24 hours

**Comment by Invy55** [comment]
Is it possible to test zoom without a school account, because I would like to help

**Comment by 1337w0rm** [comment]
> Is it possible to test zoom without a school account, because I would like to help

You just need a google account to attend zoom meeting. Although it's quite buggy right now.

**Comment by Aniketk561** [comment]
implementing google login in heroku is quite difficult right now...cookie method won't work as now it expires for less than 5 hours...24 hour was history...the only way to make this for general use is using some kind of anti captcha service in the bot inorder to bypass the captcha..there are many such chrome extensions and services that can be implemented...this could potentially make it lot more straightforward and solve all the problems

**Comment by vanshsantoshi** [comment]
I fixed it

You're supposed to go to zoom.us , scroll down to bottom, find "Cookie Preference", click it, slide it to "Advertising Cookies" ( It'll show on right side that the setting will let you save your login) click submit and then click close. After that head to zoom.us/signin page, log in through email only (I used my protonmail ID) and it'll log in.

And, after that, never ever head to zoom.us/signin, or else your cookie will be cleared.

Login after changing these settings, is persistent. Already tested it.
